charts/external-dns/crds/v1alpha1.yaml

```diff
@@ -101,3 +101,100 @@ spec:
     storage: true
     subresources:
       status: {}
+---
```

Comment:
The CRDs for the Helm chart should be in separate files and be formatted as described in #5479. The following command would be my suggestion to handle multiple CRDs being generated.

```shell
${CONTROLLER_GEN} object crd:crdVersions=v1 paths="./apis/..." output:crd:stdout | yamlfmt - | yq eval '.metadata.annotations |= with_entries(select(.key | test("kubernetes\.io")))' --split-exp '"./charts/external-dns/crds/" + .metadata.name + ".yaml"'
```